public/javascript/services.js

```diff
@@ -74,6 +74,15 @@ services.factory('AppFactory', function($resource) {
 });
 
 /**
+ * Enable or disable application
+ */
+services.factory('AppDisableFactory', function($resource) {
```

Comment:
@nick this can do more than just disable can't it?  This route can modify the entire application parameters.